sites/public/src/forms/filters/FilterForm.tsx

```diff
@@ -64,6 +65,10 @@ const FilterForm = (props: FilterFormProps) => {
     { value: "comingSoon", label: t("listings.comingSoon") },
   ]
 
+  const homeTypeOptions = Object.values(HomeTypeEnum)?.map((val) => {
```

Comment:
This is nbd and fine to merge over, but I don't think we need to map over this set twice, it would be marginally more performant to just map once below in the `FieldGroup`

Reply:
Ooo good catch, I cleaned this up 🦑